pom.xml

```diff
@@ -209,7 +218,7 @@
         <executions>
           <execution>
             <goals>
-              <goal>integration-test</goal>
+              <!--<goal>integration-test</goal>-->
```

Comment:
Why do you need to comment this out again? Does anyone who is working with these testing apps have to do this? It would be nice to find a command-line way to accomplish what you want without having to modify the build script every time...